src/lib/profiles/data-management/Current/LoggingManagement.cpp

```diff
@@ -529,6 +529,34 @@ LoggingManagement::LoggingManagement(nl::Weave::WeaveExchangeManager * inMgr,
     mMaxImportanceBuffer = kImportanceType_Last;
 }
 
+/**
+ * @brief
+ *  Reinitialize the non-persistent counters to values other than 0
+ *
+ * This function, much like the non-persistent counters themselves, should only
+ * be used in test code.
+ *
+ * The function will reinitialize the event ID counters to the current system
+ * time for all importance levels that use the non-persistent counters.  The
+ * counters will get reinitialized only if there were no events logged at that
+ * importance level.
+ */
+void LoggingManagement::RandomizeMonotonicEventCounters(void)
```

Comment:
Given the implementation, it might be better to rename this to "ReinitializeMonotonic..." or some such.

Reply:
done